analyzers/src/SonarAnalyzer.VisualBasic/Rules/StringConcatenationInLoop.cs

```diff
@@ -58,17 +45,8 @@ protected override bool IsExpressionConcatenation(BinaryExpressionSyntax addExpr
             SyntaxKind.DoLoopWhileBlock
         };
 
-        private static readonly ImmutableArray<SyntaxKind> simpleAssignmentKinds =
-            ImmutableArray.Create(SyntaxKind.SimpleAssignmentStatement);
-
-        private static readonly ImmutableArray<SyntaxKind> compoundAssignmentKinds =
-            ImmutableArray.Create(SyntaxKind.AddAssignmentStatement, SyntaxKind.ConcatenateAssignmentStatement);
-
-        protected override ImmutableArray<SyntaxKind> SimpleAssignmentKinds => simpleAssignmentKinds;
-
-        protected override ImmutableArray<SyntaxKind> CompoundAssignmentKinds => compoundAssignmentKinds;
-
-        protected override GeneratedCodeRecognizer GeneratedCodeRecognizer => VisualBasicGeneratedCodeRecognizer.Instance;
+        protected override bool AreEquivalent(SyntaxNode node1, SyntaxNode node2) =>
+            SyntaxFactory.AreEquivalent(node1, node2);
```

Comment:
Isn't there anything in the Facade?

Reply:
I couldn't find any. I'll implement it myself